.github/workflows/main.yml

```diff
@@ -64,13 +66,15 @@ jobs:
     timeout-minutes: 40
     env:
       # for ci/run.sh:
-      OS_NAME: ${{ startsWith(matrix.os, 'ubuntu') && 'linux' || (startsWith(matrix.os, 'macos') && 'darwin' || '') }}
+      OS_NAME: ${{ startsWith(matrix.os, 'ubuntu') && 'linux' || (startsWith(matrix.os, 'macos') && 'osx' || '') }}
       MODEL: ${{ matrix.model || '64' }}
       HOST_DMD: ${{ matrix.host_dmd }}
       N: ${{ startsWith(matrix.os, 'macos') && '3' || '2' }}
       FULL_BUILD: true
       # for coverage:
       DMD_TEST_COVERAGE: ${{ matrix.coverage && '1' || '0' }}
+      # work around https://issues.dlang.org/show_bug.cgi?id=23517
+      MACOSX_DEPLOYMENT_TARGET: '11'
```

Comment:
Currently worked around in https://github.com/dlang/dmd/blob/cae656cb89212123bdfaf524b8fd7f81a5663948/compiler/src/posix.mak#L81 and https://github.com/dlang/dmd/blob/cae656cb89212123bdfaf524b8fd7f81a5663948/compiler/test/Makefile#L147